src/react/components/window/index.js

```diff
@@ -185,3 +186,41 @@ export const showFlyout = (title, content, onClose, options = {}) => {
         bringToTop
     };
 };
+
+
+/**
+ * 
+ * @param {String} type Antd Alert type
+ * @param {Function} onClose 
+ * @param {object} props Antd Alert props 
+ * @returns 
+ */
+export const showBanner = (type, onClose, props) => {
+    const element = createTmpContainer();
+    const removeWindow = createRemoveFn(element, onClose);
+    const bringToTop = createBringToTop(element);
+
+    const render = (props) => {
+        ReactDOM.render(
+            <Alert
```

Comment:
Does Alert bring something to the table here or could we just use some stylized div?

Reply:
Didn't think of this before, but it might be a very good idea to have this banner-functionality here as a common component 👍

Reply:
Alert seemed like a good idea at first, but div would probably be better.